tests/test_parser.py

```diff
@@ -185,6 +185,111 @@ def test_parser_schema_violation():
+                type: string
+                key_23:
+                description: Sample extra key
+                type: string
+                key_24:
+                description: Sample extra key
+                type: string
+                key_25:
+                description: Sample extra key
+                type: string
+                key_26:
+                description: Sample extra key
+                type: string
+                key_27:
+                description: Sample extra key
+                type: string
+                key_28:
+                description: Sample extra key
+                type: string
+                key_29:
+                description: Sample extra key
+                type: string
+                key_30:
+                description: Sample extra key
+                type: string
+                key_31:
+                description: Sample extra key
+                type: string
+                key_32:
+                description: Sample extra key
+                type: string
+                key_33:
+                description: Sample extra key
+                type: string
+                key_34:
+                description: Sample extra key
+                type: string
+                key_35:
+                description: Sample extra key
+                type: string
+                key_36:
+                description: Sample extra key
+                type: string
+                key_37:
+                description: Sample extra key
+                type: string
+                key_38:
+                description: Sample extra key
+                type: string
+                key_39:
+                description: Sample extra key
+                type: string
+                key_40:
+                description: Sample extra key
+                type: string
+                key_41:
+                description: Sample extra key
+                type: string
+                key_42:
+                description: Sample extra key
+                type: string
+                key_43:
+                description: Sample extra key
+                type: string
+                key_44:
+                description: Sample extra key
+                type: string
+                key_45:
+                description: Sample extra key
+                type: string
+                key_46:
+                description: Sample extra key
+                type: string
+                key_47:
+                description: Sample extra key
+                type: string
+                key_48:
+                description: Sample extra key
+                type: string
+                key_49:
+                description: Sample extra key
+                type: string
+                key_50:
+                description: Sample extra key
+                type: string
+                key_51:
+                description: Sample extra key
+                type: string
```

Comment:
At what point should we generate this? :D (all good, keep as is for now)

Reply:
The thought did cross my mind when I had to change the second test sample dataset.  🙃